guides/common/modules/ref_best-practices-for-products-and-repositories.adoc

```diff
@@ -0,0 +1,29 @@
+[id="best-practices-for-products-and-repositories_{context}"]
+= Best practices for products and repositories
+
+* Use one content type per product and content view, for example {client-content-type} content only.
+* Use file repositories for installation media.
+File repositories require a `PULP_MANIFEST` file that you can create using `pulp-manifest /path/to/files`.
+Use local repositories with `\file:///path/to/files` as *Upstream URL* in {Project}, for example for installation media.
```

Comment:
IMHO local file repositories are a worst practice, not a best practice. They're only good for one time imports. We're heading to a networked world and if we look ahead, then we see highly available deployments (often asked for by customers). Then local repositories are a nightmare. I also have a hard time seeing how they're work in container based deployments such as Kubernetes/OpenShift.

Your alternative is IMHO more of a best practice.

Reply:
Using synced content does not work for various non-RHEL operating systems. I suggest to make it "orcharhino-only" until this changes. Would that be OK for everyone?

Reply:
Set to "orcharhino only".